src/RobustLeastSquares.jl

```diff
@@ -1,5 +1,307 @@
 module RobustLeastSquares
 
+using StatsBase
+using IterativeSolvers
+using Logging
+
+export reweighted_least_squares, mestimator_mad
+export MEstimator, MEstimatorConvex, MEstimatorNonConvex
+export L2Estimator, L1Estimator, L1L2Estimator, HuberEstimator, FairEstimator, CauchyEstimator, SubsetEstimator, SubsetEstimatorConvex, DualEstimator, DualEstimatorConvex
+
+
+
+using PyCall
+@pyimport pyamg
+@pyimport scipy.sparse as scipy_sparse
+
+# Helper function for python matrices
+function py_csc(A::SparseMatrixCSC)
+    # create an empty sparse matrix in Python
+    Apy = scipy_sparse.csc_matrix(size(A))
+    # write the values
+    Apy[:data] = copy(A.nzval)
+    # write the indices
+    Apy[:indices] = A.rowval - 1
+    Apy[:indptr] = A.colptr - 1
+    return Apy
+end
+py_csr(A::SparseMatrixCSC) = py_csc(A)[:tocsr]()
```

Comment:
So no Algebraic Multigrid (AMG) solver?
I was wondering the same... since I'm not actually using it.